lang-python/pyside6/autobuild/defines

```diff
@@ -1,11 +1,15 @@
 PKGNAME=pyside6
 PKGSEC=libs
-PKGDEP="qt-6 python-3 llvm"
-BUILDDEP="sphinx patchelf wheel setuptools"
+PKGDEP="qt-6 python-3 llvm llvm-runtime"
+BUILDDEP="sphinx"
 PKGDES="Python bindings for the Qt 6 cross-platform application and UI framework"
 
 PKGPROV="shiboken6"
 
-# Note: The setup.py script supplied by this package's source is really
-# a wrapper for a series of custom build procedures.
-ABTYPE=self
+ABTYPE=cmakeninja
+
+CMAKE_AFTER="-DUSE_PYTHON_VERSION=3 \
+             -DBUILD_TESTS=OFF \
+             -DCMAKE_BUILD_TYPE=None \
+             -DFORCE_LIMITED_API=no \
+             -DNO_QT_TOOLS=yes"
```

Comment:
```suggestion
CMAKE_AFTER=(
	"-DUSE_PYTHON_VERSION=3"
	"-DBUILD_TESTS=OFF"
	"-DCMAKE_BUILD_TYPE=None"
	"-DFORCE_LIMITED_API=no"
	"-DNO_QT_TOOLS=yes"
)
```

Please use arrays instead of strings.